### DISCUSSION TITLE
Currency/Money Format

### DISCUSSION THREAD

**Discussion by babikriuk** [filamentphp/filament#9499] [Help]
where can I find the settings for the currency/money format used in the table builder? my code 
Tables\Columns\TextColumn::make('price')
                    ->money('IDR')
it show in table IDR 50,000.00, how do i change IDR to Rp ?

**Comment by sinnbeck** [comment]
It just uses the built in NumberFormatter class and then calls this https://www.php.net/manual/en/numberformatter.formatcurrency.php

https://github.com/filamentphp/support/blob/3.x/src/helpers.php#L19

**Reply by babikriuk** [reply]
thank you

**Reply by babikriuk** [reply]
thank you

**Comment by leandrocfe** [comment]
a workaround

```php
use Illuminate\Support\Str;

TextColumn::make('price')
    ->formatStateUsing(function ($state) {
        return Str::replace('IDR', 'Rp', format_money($state, 'IDR'));
    })
```

**Reply by babikriuk** [reply]
thank you

**Reply by babikriuk** [reply]
I tried using your method but the function format_money is not found, what should I do?

**Reply by leandrocfe** [reply]
what is the filament version on the dashboard page?

**Reply by babikriuk** [reply]
filament 3.0.91

**Reply by leandrocfe** [reply]
It should exist https://github.com/filamentphp/filament/blob/3.x/packages/support/src/helpers.php

**Reply by babikriuk** [reply]
![Screenshot 2023-11-08 094335](https://github.com/filamentphp/filament/assets/147358982/35c6a3e0-8379-47f0-bcf2-e63cc276936a)

**Reply by babikriuk** [reply]
it turns out that we must include use function Filament\Support\format_money; . it works fine now.. thank you

**Reply by nuazsa** [reply]
You only need to add a function like this to retrieve currency formatting for Indonesian.

![code](https://github.com/user-attachments/assets/abb1bb74-2e5d-4b1c-adbc-27b1ca6a8368)

**Comment by mrandyyp** [comment]
How can I do this in form layout. I wanna if typing 1000 it turns to 1.000
